spec.html

```diff
@@ -25133,66 +25133,74 @@
       <!-- es6num="20.2.1.1" -->
       <emu-clause id="sec-math.e">
         <h1>Math.E</h1>
-        <p>The Number value for _e_, the base of the natural logarithms, which is approximately 2.7182818284590452354.</p>
+        <p>The Number value for _e_, the base of the natural logarithms.</p>
         <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+        <emu-note>`Math.E` can be accurately reprented by the ECMAScript Number literal `2.7182818284590452354`.</emu-note>
       </emu-clause>
 
       <!-- es6num="20.2.1.2" -->
       <emu-clause id="sec-math.ln10">
         <h1>Math.LN10</h1>
-        <p>The Number value for the natural logarithm of 10, which is approximately 2.302585092994046.</p>
+        <p>The Number value for the natural logarithm of 10.</p>
         <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+        <emu-note>`Math.LN10` can be accurately reprented by the ECMAScript Number literal `2.302585092994046`.</emu-note>
```

Comment:
`s/reprented/represented/g`

`ECMAScript` seems meta here. Can we only use `represented by 2.302585092994046` or at least `represented by the Number 2.302585092994046`?